RecoVertex/V0Producer/src/V0Fitter.cc

```diff
@@ -225,6 +249,10 @@ void V0Fitter::fitAll(const edm::Event& iEvent,
       double sigmaDistMagXY = sqrt(ROOT::Math::Similarity(totalCov, distVecXY)) / distMagXY;
       if (distMagXY / sigmaDistMagXY < vtxDecaySigXYCut_)
         continue;
+      if (distMagXY < vtxDecayXYCut_)
+        continue;
+      if (posTransTkPtr->charge() * negTransTkPtr->charge() > 0 && distMagXY < ssVtxDecayXYCut_)
+        continue;
```

Comment:
Avoid computation of `sigmaDistMagXY` if not needed:
```suggestion
      if (distMagXY < vtxDecayXYCut_)
        continue;
      if (posTransTkPtr->charge() * negTransTkPtr->charge() > 0 && distMagXY < ssVtxDecayXYCut_)
        continue;
      double sigmaDistMagXY = sqrt(ROOT::Math::Similarity(totalCov, distVecXY)) / distMagXY;
      if (distMagXY < vtxDecaySigXYCut_ * sigmaDistMagXY)
        continue;
```